Add SharingDomainRestrictionMode examples (#496)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

This example sets Manual Version Storage Limits on all new document libraries at Tenant Level by limiting the number of major versions with no time limits.

### EXAMPLE 14

```powershell
Set-SPOTenant -SharingDomainRestrictionMode "AllowList" -SharingAllowedDomainList "contoso.com"
```

> This example sets that users will be able to share with external collaborators coming only from that email domain.

Reviewer: ```suggestion
This example enables users to share with external collaborators from that email domain only.
```

### EXAMPLE 15

```powershell
Set-SPOTenant -SharingDomainRestrictionMode "BlockList" -SharingBlockedDomainList "contoso.com"
```

> This example sets that users will be able to share with all external collaborators apart from the ones on the BlockedDomainList.

Reviewer: ```suggestion
This example enables users to share with all external collaborators except for those on the BlockedDomainList.
```
